lib/sisimai/bite/email/ezweb.rb

```diff
@@ -0,0 +1,260 @@
+              next
+            end
+          end
+
+          if readcursor & Indicators[:'message-rfc822'] > 0
+            # After "message/rfc822"
+            if e.empty?
+              blanklines += 1
+              break if blanklines > 1
+              next
+            end
+            rfc822list << e
+
+          else
+            # Before "message/rfc822"
+            next if readcursor & Indicators[:deliverystatus] == 0
+            next if e.empty?
+
+            # The user(s) account is disabled.
+            #
+            # <***@ezweb.ne.jp>: 550 user unknown (in reply to RCPT TO command)
+            #
+            #  -- OR --
+            # Each of the following recipients was rejected by a remote
+            # mail server.
+            #
+            #    Recipient: <******@ezweb.ne.jp>
+            #    >>> RCPT TO:<******@ezweb.ne.jp>
+            #    <<< 550 <******@ezweb.ne.jp>: User unknown
+            v = dscontents[-1]
+
+            if cv = e.match(/\A[<]([^ ]+[@][^ ]+)[>]\z/) ||
+                    e.match(/\A[<]([^ ]+[@][^ ]+)[>]:?(.*)\z/) ||
+                    e.match(/\A[ \t]+Recipient: [<]([^ ]+[@][^ ]+)[>]/)
+
+              if v['recipient']
+                # There are multiple recipient addresses in the message body.
+                push dscontents << Sisimai::Bite.DELIVERYSTATUS
+                v = dscontents[-1]
+              end
+
+              r = Sisimai::Address.s3s4(cv[1])
+              if Sisimai::RFC5322.is_emailaddress(r)
+                v['recipient'] = r
+                recipients += 1
+              end
+
+            elsif cv = e.match(/\A[Ss]tatus:[ ]*(\d[.]\d+[.]\d+)/)
+              # Status: 5.1.1
+              # Status:5.2.0
+              # Status: 5.1.0 (permanent failure)
+              v['status'] = cv[1]
+
+            elsif cv = e.match(/\A[Aa]ction:[ ]*(.+)\z/)
+              # Action: failed
+              v['action'] = cv[1].downcase
+
+            elsif cv = e.match(/\A[Rr]emote-MTA:[ ]*(?:DNS|dns);[ ]*(.+)\z/)
+              # Remote-MTA: DNS; mx.example.jp
+              v['rhost'] = cv[1].downcase
+
+            elsif cv = e.match(/\A[Ll]ast-[Aa]ttempt-[Dd]ate:[ ]*(.+)\z/)
+              # Last-Attempt-Date: Fri, 14 Feb 2014 12:30:08 -0500
+              v['date'] = cv[1]
+
+            else
+              next if Sisimai::String.is_8bit(e)
+              if cv = e.match(/\A[ \t]+[>]{3}[ \t]+([A-Z]{4})/)
+                #    >>> RCPT TO:<******@ezweb.ne.jp>
+                v['command'] = cv[1]
+
+              else
+                # Check error message
+                if rxmessages.find { |a| e =~ a }
+                  # Check with regular expressions of each error
+                  v['diagnosis'] ||= ''
+                  v['diagnosis']  += ' ' + e
+
+                else
+                  # >>> 550
+                  v['alterrors'] ||= ''
+                  v['alterrors']  += ' ' + e
+                end
```

Comment:
__[rubocop]__
* Style/IfInsideElse: Convert `if` nested inside `else` to `elsif`.

<!-- Comment by [SideCI](https://sideci.com) -->